antora-playbook.yml

```diff
@@ -106,13 +106,13 @@ asciidoc:
     prod-operator: che-operator
     prod-operator-image-name: che-operator
     prod-prev-ver-major: 6
-    prod-prev-ver: 7.28
+    prod-prev-ver: 7.29
     prod-short: Che
     prod-upstream: Eclipse{nbsp}Che
     prod-url: https://che-host:che-port
     prod-ver-major: 7
-    prod-ver: 7.29
-    prod-ver-patch: 7.29.1
+    prod-ver: 7.30
```

Comment:
should this be once again wrapped with quotes once https://github.com/eclipse/che-docs/pull/1979 is merged?

Reply:
Yes.